code/script.js

```diff
@@ -1,19 +1,231 @@
-// Start here
-
 // Step 1 - Welcome and introduction
-// Your code goes here
 alert(
   `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`
-)
+);
+
+// Ask for name
+let userName = prompt(`What is your name?`);
+console.log("Name:", userName);
```

Comment:
Please remove the the console.log() around the code. Generally they need to be deteled when the code has been tested and works as meant to